userguide/content/en/docs/get-started/docsy-as-module/installation-prerequisites.md

```diff
@@ -10,26 +10,49 @@ This page describes the prerequisites for building a site that uses Docsy as a H
 
 ## Install Hugo
```

Comment:
Consider mentioning installing the Hugo binary via NPM: https://deploy-preview-1597--docsydocs.netlify.app/docs/get-started/other-options/#as-an-npm-module

/cc @LisaFC